assets/js/utils/sanitize-html.ts

```diff
@@ -1,27 +1,20 @@
 /**
  * External dependencies
  */
-import DOMPurify from 'dompurify';
-
-type sanitizedHTMLObject = {
-	// eslint-disable-next-line @typescript-eslint/naming-convention
-	__html: string;
-};
+import { sanitize } from 'dompurify';
 
 const ALLOWED_TAGS = [ 'a', 'b', 'em', 'i', 'strong', 'p', 'br' ];
 const ALLOWED_ATTR = [ 'target', 'href', 'rel', 'name', 'download' ];
 
 export const sanitizeHTML = (
 	html: string,
 	config?: { tags?: typeof ALLOWED_TAGS; attr?: typeof ALLOWED_ATTR }
-): sanitizedHTMLObject => {
+) => {
 	const tagsValue = config?.tags || ALLOWED_TAGS;
 	const attrValue = config?.attr || ALLOWED_ATTR;
 
-	return {
-		__html: DOMPurify.sanitize( html, {
-			ALLOWED_TAGS: tagsValue,
-			ALLOWED_ATTR: attrValue,
-		} ),
-	};
+	return sanitize( html, {
+		ALLOWED_TAGS: tagsValue,
+		ALLOWED_ATTR: attrValue,
+	} );
```

Comment:
Based on your feedback in https://github.com/woocommerce/woocommerce-blocks/pull/7147#discussion_r980380183, I was wondering if we shouldn't remove the object keys. The final function could look something like this:

```ts
/**
 * External dependencies
 */
import { sanitize } from 'dompurify';

const DEFAULT_TAGS = [ 'a', 'b', 'em', 'i', 'strong', 'p', 'br' ];
const DEFAULT_ATTR = [ 'target', 'href', 'rel', 'name', 'download' ];

export const sanitizeHTML = (
	html: string,
	config?: { tags?: typeof DEFAULT_TAGS; attr?: typeof DEFAULT_ATTR }
) => {
	const ALLOWED_TAGS = config?.tags || DEFAULT_TAGS;
	const ALLOWED_ATTR = config?.attr || DEFAULT_ATTR;

	return sanitize( html, { ALLOWED_TAGS, ALLOWED_ATTR } );
};
```

Reply:
Method A & B work the same way, so either is ok.  I prefer method B because it's cleaner.
```
// Method A:
return sanitize( html, {
	ALLOWED_TAGS: tagsValue,
	ALLOWED_ATTR: attrValue,
} );
```

```
// Method B:
return sanitize( html, { ALLOWED_TAGS, ALLOWED_ATTR } );
```

Reply:
> I prefer method B because it's cleaner.

I agree. Method B indeed looks much cleaner.